test/regress/regress0/sygus/hd-19-d1-prog-dup-op.sy

```diff
@@ -0,0 +1,32 @@
+; EXPECT: unsat
+; COMMAND-LINE: --cegqi-si=all --sygus-out=status
+
+(set-logic BV)
+
+(define-fun hd19 ((x (BitVec 32)) (m (BitVec 32)) (k (BitVec 32))) (BitVec 32) 
```

Comment:
These are regressions, the formatting rules shouldn't be applied to these.

Reply:
I think that trailing spaces are bad enough that they do not get to exist anywhere.

Reply:
Regressions are meant to test user inputs. Having this rule would be equivalent to saying that we expect users to remove trailing space from all inputs they give to cvc4.

Reply:
I'm complaining because the file is living inside the CVC4 repo under version control. Any such file is amenable to one of the (many) evils of trailing spaces: creating spurious diffs when somebody with a setting that automatically deletes trailing spaces modifies such a file (e.g. for adding an option).

Reply:
I don't think diffs on regressions is a big deal, and regardless this should be avoided by using the proper setting on the editor.
The more important thing is that our regressions should not be biased. It is useful for the regressions have trailing spaces because this tests the behavior of cvc4 on benchmarks we see in the wild.